drivers/storage/portworx/podsecuritypolicies.go

```diff
@@ -0,0 +1,96 @@
+package portworx
+
+import (
+	"github.com/libopenstorage/operator/pkg/constants"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func portworxPodSecurityPolicies() []policyv1beta1.PodSecurityPolicy {
```

Comment:
We should create a component for PSPs instead of having it here. That way the Reconcile and Delete loops will automatically handle creation and removal of these PSPs.

Reply:
Added this separately to be sure that psp is created before other resources as components aren't ordered.
Create a component for PSP.